app-crypt/moolticute/moolticute-0.30.8.ebuild

```diff
@@ -0,0 +1,67 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+if [[ ${PV} == 9999* ]]; then
+	EGIT_REPO_URI="https://github.com/mooltipass/moolticute.git"
+	inherit git-r3
+	KEYWORDS=""
```

Comment:
Please don't duplicate `KEYWORDS` lines. No need to unset it for 9999 anyway.

Reply:
Thanks for your review @a17r! I wasn't aware of this rule. Now fixed :)